src/app/lightning/get-balances.ts

```diff
@@ -6,11 +6,11 @@ import { ErrorLevel } from "@domain/shared"
 import { TxDecoder } from "@domain/bitcoin/onchain"
 
 import { LndService } from "@services/lnd"
-import { LocalCacheService } from "@services/cache"
+import { RedisCacheService } from "@services/cache"
 import { OnChainService } from "@services/lnd/onchain-service"
 import { recordExceptionInCurrentSpan } from "@services/tracing"
 
-const cache = LocalCacheService()
+const cache = RedisCacheService()
```

Comment:
I'm assuming exporter has access to redis based on this: https://github.com/GaloyMoney/charts/blob/f5ba5c9ce6914a6c2e378e9c46c10f5e8ac4c955/charts/galoy/templates/exporter-deployment.yaml#L83